Cargo.toml

```diff
@@ -2,6 +2,22 @@
 resolver = "2"
 members = [ "crates/*"]
 
+default-members = [
+  "crates/allocator",
+  "crates/bindings",
+  "crates/build",
+  # Deliberately exclude the `control` crate from top level cargo commands.
```

Comment:
Why exclude it? Would it cause a problem with `cargo test`?

Reply:
Yeah. Right now, our Makefile and CI for the flow repo all assume that the artifact we're building is `flowctl`. This is fine, but `control` is a different build artifact. It's tests will require a postgres connection. I didn't want to inject that into the main `flowctl` build. I assume we'll have similar concerns for the other Rust binaries we're moving into the flow repo. This felt like a safe move for now since it doesn't upset the way `flowctl` is currently built - that seems like a potentially deep rabbit hole. All of this is also why I added a separate CI job for `control`.

Take all that as weakly held motivation for why I did things this way. I don't expect things to necessarily stay this way.

Reply:
Ahh that all makes sense, and feels like important context for whoever ends up working on improving the situation. Might be a good candidate for a comment.